cypress/integration/advanced-functionalities/admin/system/messaging-queues/Un-delete.spec.js

```diff
@@ -0,0 +1,34 @@
+/// <reference types="cypress" />
+const adminURL = Cypress.env('ADMIN_URL')
+const email = Cypress.env('USER_EMAIL')
+const password = Cypress.env('USER_PASSWORD')
+
+describe('Test for un-deleting a messaging queue', () => {
+  before(() => {
+    if (!window.sessionStorage.getItem('auth.refresh-token')) {
+      cy.login({ email, password, url: adminURL })
+    }
+  })
+
+  context('Test for un-deleting a messaging queue', () => {
+    it('should be able to un-delete a messaging queue', () => {
+      cy.visit(adminURL + '/')
+      // We wait 3s in order the page to be fully loaded
+      cy.wait(3000)
+      cy.get('.nav-sidebar').contains('Messaging Queues').click()
+      // We wait 1s in order the page to be fully loaded
+      cy.wait(1000)
+      cy.get('[data-test-id="filter-deleted-queues"]').contains('Only').click()
+      cy.get('[data-test-id="input-search"]').type('TestQueue')
+      cy.get('#resource-list > tbody > tr:last > td:last > a').click()
+      cy.get('[data-test-id="input-deleted-at"]').should('exist')
+      cy.get('[data-test-id="button-undelete"]').click()
```

Comment:
Check if:
- delete btn isn't shown
- createdAt and updatedAt are shown
- new and submit btns are shown